src/dymaptic.GeoBlazor.Core/Objects/ArealUnit.cs

```diff
@@ -22,6 +23,58 @@ public enum ArealUnit
     SquareFeet,
     SquareMeters,
     SquareYards,
-    SquareKilometers
+    SquareKilometers,
+    SquareMiles
+#pragma warning restore CS1591
+}
+
+/// <summary>
+/// Units for area measurement. Use one of the possible values listed below or any of the numeric codes for area units.
+///  <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-core-units.htm">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+[JsonConverter(typeof(EnumToKebabCaseStringConverter<AreaUnit>))]
+public enum AreaUnit
```

Comment:
We need a better way to identify the units used in GeometryEngine vs. the ones used in the new widget. Since they changed the names on me anyways, could we create a new `GeometryEngineAreaUnit`, create overloads in `GeometryEngine.cs` that take those instead, and mark the ones that take `ArealUnit` as `[Obsolete]`?